### DISCUSSION TITLE
Error by composer install

### DISCUSSION THREAD

**Discussion by homeoftheunits** [spatie/laravel-medialibrary#2718] [Q&A]
Hey Guys,

i want to install medialibrary in my laravel 8 app, running on php 8.1

I got this error:
```
Type of Spatie\MediaLibrary\MediaCollections\Models\Media::$casts must be array (as in class Illuminate\Database\Eloquent\Model)
```

Any ideas?

Cheers
Ralf

**Comment by erikn69** [comment]
Version of the package?
`Models\Media::$casts` is an array, maybe you broke the implementation
https://github.com/spatie/laravel-medialibrary/blob/47807503f79bd5a4385b70a6053fb886dfa344d1/src/MediaCollections/Models/Media.php#L45-L50
